integration-tests/src/test/resources/features/error-handling.feature

```diff
@@ -125,3 +125,54 @@ Feature: Error handling tests
     }
     """
     Then Webhook site with id "${env.webhook.site.uuid}" contains request with text ""id":"${cloud-event.error-handling-update-test.id}","source":"mySource","type":"myType","subject":"mySubject"" within 1 minute
+
+  @errorhandlingdlqheaders
+  Scenario: A failure occurs and a cloud event message is sent by managed connector to DLQ with correct headers
+    Given authenticate against Manager
+    And create a new Bridge with body:
+    """
+    {
+        "name": "ehBridge",
+        "cloud_provider": "aws",
+        "region": "us-east-1",
+        "error_handler": {
+            "type": "webhook_sink_0.1",
+            "parameters": {
+                "endpoint": "https://webhook.site/${env.webhook.site.uuid}"
+            }
+        }
+    }
+    """
+    And the Bridge "ehBridge" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "ehBridge" is available within 2 minutes
+
+    And add a Processor to the Bridge "ehBridge" with body:
+    """
+    {
+       "name": "sqsProcessor",
+       "source": {
+          "type": "aws_sqs_source_0.1",
+          "parameters": {
+                "aws_queue_name_or_arn": "${env.aws.queue-name-or-arn}",
+                "aws_region": "${env.aws.region}",
+                "aws_access_key" : "${env.aws.access-key}",
+                "aws_secret_key" : "${env.aws.secret-key}"
```

Comment:
Would it be possible to supply the value using resolver?
Something like https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/context/resolver/SlackChannelNameResolver.java

That way you can avoid duplication of parameters and pass them just using TNB config.